sci-biology/mne-python/Manifest

```diff
@@ -1 +1,3 @@
-DIST v0.7.zip 58120981 SHA256 26711803c8068adc7f3c06997b6bd1e076fea170e9f87af3b488c51eb6e3caa9 SHA512 1221038daa9687e358c7b51aed969c0174e5ea979e35c2e2663a5d2795b751f837bb8e1f45fd12251f713b6b69a5c5086e8f07d417e23abfa4302562f271f774 WHIRLPOOL a44c067c02270285103ad978d2d1afbf46972ad39eeec77548d78bb1515bd4a22fca713c2e814d5b462b09ce3caad2696deda1b09f67ed9f3c8f0a74de3fdaee
+DIST mne-python-0.11.zip 59598372 SHA256 44e1f0b191ae4b7ba90255a80ce2d5aa2858f922d13267608616348a150334e9 SHA512 cb94d8dabab9e7625993f108a7c1c42ad4c43e0f59c19a1afb948eeec1adcbe52f310cad591bcc1212c8660ea2ee31422342b6d70e0aae3359b33f89358b19a3 WHIRLPOOL 20cae9ac9098ce5783aa1d32f14946ab89c20ee06d0bba8ba3ac1377164cee6e922c7c9577c9f07c568944463863c91f5fffdd702e23368c80276619071919c9
+DIST mne-python-0.12.zip 59738556 SHA256 c72aa7082787206fcfd6462c43d0821b4937b4d1c69561fc28e3c69d57f2a44b SHA512 7d41926d393e0fb2a4c4c3b643803c96f6797385059370fcf43b06756f0bfed4311f8b6c0d342e47e3589bf349c521f4fed67aeb6a1a6d349190e00db40edd96 WHIRLPOOL 568c06d7db6cc8bc18c12eaa6b067797bed852b8973adaa3b29912f9ea9f142da19fdb811f9bc7d8079dedd3b2d1849eb753ebfddb876c0214c2c242113c90fe
+DIST v0.7.zip 58120981 SHA256 b2f89345690989db748fd5bc4c07762b7134e99abc5634facca5a1ee4b32fde4 SHA512 72fe588787febe1804fa853b454ed3a3f9383c93b53052bc5cc2ecf01b85e9f44550fecc91a7a5c26b52853e3a43e0b0cf9ae93c8d145ee50c6fd33b22a6fb9e WHIRLPOOL 3a464a88ec91b8d7d433f71e7814fe992b78e016e63ac42a77b8e0ee729ea409994432698f0583230f54a508e92b2b40312f0c8aca7fd1f830175110abe7ca9c
```

Comment:
This is not an addition that I made - the file name is the same for the current mne-python ebuild by @mluessi - which has been in this overlay for a few years now. I would rather keep his ebuild as is.